src/main/scala/EmrSparkPlugin.scala

```diff
@@ -49,6 +54,7 @@ object EmrSparkPlugin extends AutoPlugin {
     val sparkS3LoggingFolder = settingKey[Option[String]]("S3 folder for application's logs")
     val sparkS3JsonConfiguration = settingKey[Option[String]]("S3 location for the EMR cluster json configuration")
     val sparkAdditionalApplications = settingKey[Option[Seq[String]]]("Applications other than Spark to be deployed on the EMR cluster, these are case insensitive.")
+    val sparkEMRSteps = settingKey[Option[Seq[StepConfig]]]("Multiple steps to run once cluster is setup")
```

Comment:
Hmmmm, I see. I think we can keep it like this now, but I also feel that it's not convenient enough.
Could you give an example on how you create the steps in `build.sbt`?
Maybe I can work on some ideas, too.